AndroidSDKCore/src/main/java/com/leanplum/internal/Constants.java

```diff
@@ -144,6 +144,7 @@ public static boolean isNoop() {
     public static final String IAP_CURRENCY_CODE = "currencyCode";
     public static final String IAP_ITEM = "item";
     public static final String INCLUDE_DEFAULTS = "includeDefaults";
+    public static final String INCLUDE_CONTENT_ASSIGNMENTS = "includeContentAssignments";
```

Comment:
Isn't this supposed to be `includeVariantDebugInfo` as per design doc: https://leanplum.atlassian.net/wiki/spaces/PRODDEV/pages/548503556/Tinder+Expose+granular+information+on+variables+and+A+B+tests+to+SDK